src/pages/CreateSalary/CreateSalary.scss

```diff
@@ -1,8 +1,40 @@
-.modal {
-  color: red;
-  .input {
-    background: #ded4d440;
-    border-radius: 5px;
-    --padding-start: 10px;
+ion-content {
+  .form-list {
+    .input-wrapper {
+      --padding-start: 0;
```

Comment:
just curious, what's this CSS sorcery? XD

Reply:
This are CSS variables, a way to replace/override any value that Ionic has by default. If you try to apply a normal CSS it won't be effective due to the shadow-dom of the Ionic components (main intention on this is tp avoid CSS leaking)